src/core/lib/okp4/okp4.service.ts

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Let's not use it globally for file, just add over places where you need `any` explicitly